libraries/botbuilder-dialogs-adaptive/src/luis/luisAdaptiveRecognizer.ts

```diff
@@ -94,4 +91,59 @@ export class LuisAdaptiveRecognizer extends Configurable implements Recognizer {
         }
         return options;
     }
+    /**
+     * Fills the event properties for LuisResult event for telemetry.
+     * These properties are logged when the recognizer is called.
+     * @param recognizerResult Last activity sent from user.
+     * @param telemetryProperties Additional properties to be logged to telemetry with the LuisResult event.
+     * @param dialogContext Dialog context.
+     * @returns A dictionary that is sent as properties to BotTelemetryClient.trackEvent method for the LuisResult event.
+     */
+    protected  fillRecognizerResultTelemetryProperties(recognizerResult: RecognizerResult, telemetryProperties: { [key: string]: string }, dialogContext: DialogContext): { [key: string]: string } {
+        var logPersonalInfoVnE = this.logPersonalInformation.tryGetValue(dialogContext.state);
+        var logPersonalInfo = logPersonalInfoVnE.value;
+        var error = logPersonalInfoVnE.error;
```

Comment:
Just ignore error values that has never been used.
const { logPersonalInfo } = this.logPersonalInformation.tryGetValue(dialogContext.state);